gobblin-test-harness/src/main/java/gobblin/test/execution/operator/SetupOperator.java

```diff
@@ -1,5 +1,4 @@
-/*
- * Copyright (C) 2014-2015 LinkedIn Corp. All rights reserved.
+/* (c) 2014 LinkedIn Corp. All rights reserved.
```

Comment:
Check out: https://help.github.com/articles/syncing-a-fork/ for directions on how to do this.